File: ncms-engine/ncms-engine-core/src/main/java/com/softmotions/ncms/adm/AdmModule.java
package com.softmotions.ncms.adm;

import com.google.inject.AbstractModule;
import com.google.inject.Singleton;

/**
 * Admin-zone GUI RESTFull modules.
 *
 * @author Adamansky Anton (adamansky@gmail.com)
 */
public class AdmModule extends AbstractModule {

    @Override
    protected void configure() {
        bind(WorkspaceRS.class).in(Singleton.class);
        bind(AdmUIResourcesRS.class).in(Singleton.class);
    }
}

File: ncms-engine/ncms-engine-core/src/main/java/com/softmotions/ncms/adm/AdmModule.kt
package com.softmotions.ncms.adm

import com.google.inject.AbstractModule
import com.google.inject.Singleton

/**
 * Admin-zone GUI RESTFull modules.
 * @author Adamansky Anton (adamansky@softmotions.com)
 */
class AdmModule : AbstractModule() {

    override fun configure() {
        bind(WorkspaceRS::class.java).`in`(Singleton::class.java)
        bind(AdmUIResourcesRS::class.java).`in`(Singleton::class.java)
    }
}
